packages/registry/src/contract-addresses.ts

```diff
@@ -0,0 +1 @@
+export const ENTRYPOINT = '0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789'
```

Comment:
Can we assume that this address is deployed on the same address for all evm equivalent chains?

If not then looks like we can manage using `chainId` later

Reply:
It is the same on all chains